dashboard/src/shared/Auth.ts

```diff
@@ -33,13 +33,17 @@ export class Auth {
     };
   }
 
-  // Throws an error if the token is invalid
-  public static async validateToken(token: string) {
+  // Throws an error if accessing the Kubernetes API returns an Unauthorized error
+  public static async validate(token?: string) {
```

Comment:
this is the main change, in the case of having an available OIDC proxy the token will be optional so a request to `/api/kube` will success without us having to worry about the token.